packages/main/src/Input.js

```diff
@@ -520,7 +521,7 @@ class Input extends UI5Element {
 		this.suggestionSelectionCanceled = false;
 
 		// Indicates if the change event has already been fired
```

Comment:
sounds like a boolean, but actually this is a string value

Reply:
It used to be a boolean, but now is a string and the name has been changed to **__changeFiredValue_**. Any suggestions are welcomed :)